### ISSUE TITLE
Get rid of Glide

### ISSUE THREAD

**Issue by leonlatsch** [leonlatsch/Photok#318]
After #317 the thumbnail creation is the only place missing that uses glide.

Replace with coil and remove glide from the app

**Comment by starry-shivam** [comment]
Can I work on this? If this isn't assigned to anybody yet, nice app btw. I'd also love to see material you/dynamic colors support, but maybe I should create a separate issue for that.

**Comment by leonlatsch** [comment]
Sure, go ahead.

What would be needed is inside the PhotoRepository. Glide is used to create thumbnails from full size ByteArrays. The goal is to remove glide from there and replace it with coil. Also, if you want, you could wrap this logic in a more specialized class.
Also if you are done, feel free to remove the glide dependency then :)

Thank you :)

**Comment by ignoramous** [comment]
*rethinkdns dev here*

We use *Glide*.

> The goal is to remove glide from there and replace it with coil. 

Is it because *Coil* is more performant and efficient? If so, we'd want to make an effort to switch too. Our usecase is mainly 2-tier caching (memory/disk) downloaded favicons.

**Comment by leonlatsch** [comment]
So, I have heard that coil is more performant than glide from coworkers. But I don't know anything about this topic.

I have used coil in the new gallery that is written in jetpack compose, because I knew its ready for compose and supported custom  image loading for the encrypted files.

Since this worked great and I could also remove another usage of glide in #317 that now uses the same code as in the gallery. I didn't want to have glide in the project alongside coil.
So this is just for the sake of not having both in the project.

**Comment by leonlatsch** [comment]
Bth @starry-shivam I will review your PR once I find the time. Been busy lately with work, etc.
